src/realm/object-store/sync/app.cpp

```diff
@@ -1080,7 +1157,7 @@ void App::do_authenticated_request(Request&& request, const std::shared_ptr<Sync
                                       std::move(completion));
         }
         else {
-            completion(std::move(response));
+            completion(response);
```

Comment:
Why did we stop moving the response here and below?

Reply:
Because the Response is passed to the completion handlers as a const& instead of an r-value.